src/magql/filter.py

```diff
@@ -181,6 +187,10 @@ def condition(filter_value, filter_operator, field):
             return field.like(f"%{filter_value}%")
         elif filter_operator == "EQUALS":
             return field == filter_value
+        elif filter_operator == "EXISTS":
+            return field.like("%")
+        elif filter_operator == "DOESNOTEXIST":
+            return field == None  # noqa: E711
```

Comment:
Have you tried `field.is(None)`

Reply:
I just tried it. Invalid Syntax.
![image](https://user-images.githubusercontent.com/33336510/91874747-7348f280-ec2f-11ea-8da3-3683df72efd2.png)
Ah, I see I was wrong. It's not evaluated on the front end... I see the graphQL request to the back-end. When I was trouble shooting, if I would print(Model.field == None),  I would see Model.field IS NULL in the terminal, so I was assuming the Python was compiled to JavaScript to be evaluated in browser, but now I see that's not the case.
I'll look into a better solution again today on this one.

Reply:
The SQLAlchemy `is_()` operator has a trailing underscore to distinguish it from the Python keyword.

Reply:
That fixed it. Thanks David!